public/src/models.ts

```diff
@@ -13,25 +13,22 @@ export interface IUser {
 
 export interface IAssignmentLink {
     /**
-     * The course to the group
+     * The current course
```

Comment:
I think it looks better to use `// Comment...` instead of `/** Comment ... */` when in structs like this, since it takes up less space. Also, not sure we need a comment on all fields here, e.g. `The current course` could either be removed or made explicit by naming it `currentCourse: Course;` in the struct/interface. But it is ok to keep the comment, if it makes sense to have a comment for the other fields (and it does, I think for the other fields...)

Reply:
Done.